build.gradle

```diff
@@ -29,8 +37,17 @@ dependencies {
 
 group = 'com.gimp'
 version = '2.0'
-sourceCompatibility = '1.8'
+sourceCompatibility = isGimpDev ? '11' : '8';
```

Comment:
does 8 == 1.8?

Reply:
Ohhhh that should be `1.8`, let me fix that (but yeah technically it's the same thing)